rust/operator-binary/src/kafka_controller.rs

```diff
@@ -788,31 +793,18 @@ fn build_broker_rolegroup_statefulset(
         .add_volume_and_volume_mounts(&mut pod_builder, &mut cb_kcat_prober, &mut cb_kafka)
         .context(AddVolumesAndVolumeMountsSnafu)?;
 
-    cb_get_svc
-        .image_from_product_image(resolved_product_image)
-        .command(vec![
-            "/bin/bash".to_string(),
-            "-x".to_string(),
-            "-euo".to_string(),
-            "pipefail".to_string(),
-            "-c".to_string(),
-        ])
-        .args(vec![kafka_security.svc_container_commands()])
-        .add_env_vars(vec![EnvVar {
-            name: "POD_NAME".to_string(),
-            value_from: Some(EnvVarSource {
-                field_ref: Some(ObjectFieldSelector {
-                    api_version: Some("v1".to_string()),
-                    field_path: "metadata.name".to_string(),
-                }),
-                ..EnvVarSource::default()
-            }),
-            ..EnvVar::default()
-        }])
-        .add_volume_mount("tmp", STACKABLE_TMP_DIR)
-        .resources(merged_config.resources.clone().into());
+    let mut pvcs = merged_config.resources.storage.build_pvcs();
 
-    let pvcs = merged_config.resources.storage.build_pvcs();
+    // bootstrap listener should be persistent,
+    // main broker listener is an ephemeral PVC instead
+    pvcs.push(
+        ListenerOperatorVolumeSourceBuilder::new(
+            &ListenerReference::ListenerName(kafka.bootstrap_service_name(rolegroup_ref)),
+            &Labels::new(),
```

Comment:
```suggestion
            &Labels::role_selector(kafka, APP_NAME, &rolegroup_ref.role)
                .context(LabelBuildSnafu)?,
```

Reply:
7aa5919b87babc2ec613025131c0448f63a995e1 applies it to the Listener itself.

Reply:
7dae34d4fa5774c3415c80de60ba604b8cffdd91 for the PVC

Reply:
Had to revert it slightly to avoid upgrade issues: 291308109e40448f238742ec7a7b0541f608d11d